### PR TITLE
React to Razor.Design removal

### PR DESCRIPTION
**Pull request by pranavkm**
* Update to feature branch versions of Razor.Sdk and Mvc
* Do not target MVC 3.0 in a netcoreapp2.x targeting project

### CODE REVIEW

File: sample/ApplicationInsightsHostingStartupSample/ApplicationInsightsHostingStartupSample.csproj [hunk 1]
```diff
@@ -1,18 +1,30 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.1;netcoreapp2.0;net461</TargetFrameworks>
+    <TargetFrameworks>netcoreapp3.0;netcoreapp2.1;netcoreapp2.0</TargetFrameworks>
```
Comment (pranavkm) [inline sample/ApplicationInsightsHostingStartupSample/ApplicationInsightsHostingStartupSample.csproj]: We don't test `net461` so this didn't seem important to list here.

File: test/ApplicationInsights.HostingStartup.Tests/LoggingTest.cs [hunk 2]
```diff
@@ -111,7 +111,7 @@ private async Task<string> RunRequest(string targetFramework, ApplicationType ap
                 {
                     PublishApplicationBeforeDeployment = true,
                     PreservePublishedApplicationForDebugging = PreservePublishedApplicationForDebugging,
-                    TargetFramework = "netcoreapp2.0",
+                    TargetFramework = targetFramework,
```
Comment (pranavkm) [inline test/ApplicationInsights.HostingStartup.Tests/LoggingTest.cs]: Fixes test scenario which was only being tested on `netcoreapp2.0` before

**Comment by pranavkm** [comment]
FYI @pakrym \ @JunTaoLuo checking this in to preempt build break

**Comment by pakrym** [comment]
LGTM

**Comment by JunTaoLuo** [comment]
👍

**Pull request merged**